platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -201,9 +203,14 @@ void updateSepTransaction(PlatformTransactionData patch, JdbcSepTransaction txn)
       case "24":
         JdbcSep24Transaction sep24Txn = (JdbcSep24Transaction) txn;
 
+        Memo memo = MemoHelper.makeMemo(patch.getMemo(), patch.getMemoType());
```

Comment:
nit: We may use static imports when possible. 

```suggestion
        Memo memo = makeMemo(patch.getMemo(), patch.getMemoType());
```